crates/bevy_reflect/src/tuple.rs

```diff
@@ -735,7 +736,13 @@ macro_rules! impl_type_path_tuple {
     };
 }
 
-all_tuples!(impl_type_path_tuple, 0, 12, P);
+all_tuples!(
+    #[doc(fake_variadic)]
+    impl_type_path_tuple,
+    0,
+    12,
+    P
+);
 
 #[cfg(feature = "functions")]
 const _: () = {
```

Comment:
Any reason not to include the `all_tuples!` usages in this block? I think it should be doable if we modify the internal macros (e.g. `impl_get_ownership!`, `impl_from_arg!`, and `impl_into_return!`).

Reply:
I assume because the block is gated by `#[cfg(feature = "functions")]`? I'm afraid that I'm the wrong person to answer this question, I saw this code for the first time :)